signer/src/config.rs

```diff
@@ -205,6 +207,15 @@ impl Validatable for SignerConfig {
     }
 }
 
+/// Configuration for the Stacks event observer server (hosted within the signer).
+#[derive(Debug, Clone, Deserialize)]
+pub struct EventObserverConfig {
+    /// The address to bind the server to.
+    pub bind: std::net::IpAddr,
+    /// The port to bind the server to.
+    pub port: u16,
```

Comment:
Why not combine them into one field and deserialize to a `std::net::SocketAddr`?

Reply:
Good, I thought I had tried that and it wasn't supported, but apparently I tried something else -- changed in [e05f368](https://github.com/stacks-network/sbtc/pull/482/commits/e05f36856afed2f4c25663fc7fe56ff6a3fece7a) :)